Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/util/HelperUtil.java

```diff
@@ -402,5 +407,25 @@ public static int getHashCodeBy(Object... values) {
         return retList;
     }
 
+    public static String readPropertyFrom(String propertyFile, String propertyName) {
+        try {
+            return PropertyAccessor.getInstance().getProperty(propertyFile, propertyName);
```

Comment:
No need to do this since we already have this https://github.com/CONNECT-Solution/CONNECT/blob/09f0dd6f75375668dbdfec5d2102192877135da9/Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/properties/PropertyAccessor.java#L99